### PR TITLE
Fix(Order tests) Add tests for the JSON order fixtures

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
The JSON order fixtures were not properly tested
### Added

- A new test for MIP-dna, balsamic and RML json-orders
- Created two new json file fixtures

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by Vince-janv** [comment]
### Pytest pass and all JSON fixture can be uploaded to the orderportal:
Screenshot 2022-09-06 at 14 30 57
Screenshot 2022-09-06 at 14 31 23
Screenshot 2022-09-06 at 14 31 42

File: tests/apps/orderform/test_json_orderform_parser.py [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
     order_form_parser = JsonOrderformParser()
-
-    # WHEN parsing the RML orderform
-    order_form_parser.parse_orderform(order_data=mip_order_to_submit)
-
-    # THEN assert that the orderform was parsed in the correct way
-    assert order_form_parser.order_name == mip_order_to_submit.get("name")
+    order_form_parser.parse_orderform(order_data=json_order)
+    order_form: Orderform = order_form_parser.generate_orderform()
+    # THEN the created Orderform should contain samples, an order type and a delivery_type
```
Comment (henrikstranneheim) [inline tests/apps/orderform/test_json_orderform_parser.py:19]:
```suggestion

    # THEN the created Orderform should contain samples, an order type and a delivery_type
```

File: tests/conftest.py [hunk 2]
```diff
@@ -303,7 +303,7 @@ def fixture_apps_dir(fixtures_dir: Path) -> Path:
     return Path(fixtures_dir, "apps")
 
 
-@pytest.fixture(name="cgweb_orders_dir")
+@pytest.fixture(name="cgweb_orders_dir", scope="session")
 def fixture_cgweb_orders_dir(fixtures_dir: Path) -> Path:
     """Return the path to the cgweb_orders dir"""
```
Comment (henrikstranneheim) [inline tests/conftest.py:308]:
```suggestion
    """Return the path to the cgweb_orders dir."""
```

File: tests/meta/orders/conftest.py [hunk 3]
```diff
@@ -26,7 +26,7 @@
 )
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
 def all_orders_to_submit(
```
Comment (henrikstranneheim) [inline tests/meta/orders/conftest.py:30]:
```suggestion
@pytest.fixture(scope="session", name="all_orders_to_submit")
def fixture_all_orders_to_submit(
```

**Review by henrikstranneheim — APPROVED** [review]
Great!

**Comment by Vince-janv** [comment]
### No need to deploy anywhere

**Pull request merged**